index.js

```diff
@@ -0,0 +1,31 @@
+const express = require('express');
+const app = express();
+const path = require('path');
+
+app.use(express.json())
+
+app.get('/', (req, res) => {
+    res.status(200);
+    res.json({message: 'Welcome to the root URL of the server'})
+})
+
+app.get('/hello', (req, res) => {
```

Comment:
Please add these single quoted strings to constants

Reply:
Same